app/templates/_liquibase.gradle

```diff
@@ -17,7 +17,7 @@ task liquibaseDiffChangelog(dependsOn: compileJava, type: JavaExec) {
   args "--referenceUrl=hibernate:spring:<%=packageName%>.domain?dialect=<% if (devDatabaseType == 'mysql') { %>org.hibernate.dialect.MySQLInnoDBDialect<% } else if (devDatabaseType == 'postgresql') { %>org.hibernate.dialect.PostgreSQL82Dialect<% } else if (devDatabaseType == 'h2Disk') { %>org.hibernate.dialect.H2Dialect<%} %>"
   args "--username=<% if (devDatabaseType == 'mysql') { %>root<% } else if (devDatabaseType == 'postgresql') { %><%= baseName %><% } %>"
   args "--password="
-  args "--url=<% if (devDatabaseType == 'mysql') { %>jdbc:mysql://localhost:3306/<%= baseName %><% } else if (devDatabaseType == 'postgresql') { %>jdbc:postgresql://localhost/<%= baseName %><% } else if (devDatabaseType == 'h2Disk') { %>jdbc:h2:tcp://localhost:9092/./target/h2db/db:jhipster<% } %>"
+  args "--url=<% if (devDatabaseType == 'mysql') { %>jdbc:mysql://localhost:3306/<%= baseName %><% } else if (devDatabaseType == 'postgresql') { %>jdbc:postgresql://localhost/<%= baseName %><% } else if (devDatabaseType == 'h2Disk') { %>jdbc:h2:file:./target/h2db/db/<%= baseName %><% } %>"
```

Comment:
Have a look at https://github.com/jhipster/generator-jhipster/blob/master/app/templates/src/main/resources/config/_application-dev.yml#L39

I think we should use lowercase